javascript/presigned-url-example.ts

```diff
@@ -0,0 +1,140 @@
+import {MomentoSigner, CacheOperation} from '@gomomento/sdk';
+import fetch, {Response} from 'node-fetch';
+
+const signingKey = process.env.MOMENTO_SIGNING_KEY;
+if (!signingKey) {
+  throw new Error('Missing required environment variable MOMENTO_SIGNING_KEY');
+}
+const signingEndpoint = process.env.MOMENTO_ENDPOINT || '';
+if (signingEndpoint === '') {
```

Comment:
nit: I would just make this `if(!signingEndpoint)`. An empty string is falsey in javascript. And then above this you could just have `const signingEndpoint = process.env.MOMENTO_ENDPOINT;`

Reply:
That's what I had initially tried but then ended up with the error

```
Argument of type 'string | undefined' is not assignable cache to parameter of type 'string'.
 Type 'undefined' is not assignable to type 'string'. 
```
when trying to use it at like 
https://github.com/momentohq/client-sdk-examples/blob/44d1dc8767e7dbe902f97cdea1f580be831a14c6/javascript/presigned-url-example.ts#L39

I was able to get around this error with this `|| ''`  - something about how its doing the type checking? But I agree that it seems gross, especially to have in example code.

Other option seems to be pass it into the examples so it would end up being like

```
await runPresignedUrlExample(signer, signingEndpoint)
```

for whatever reason that doesn't have this error.

Reply:
a couple of ideas to consider:

1. Add a small fn called `requireEnvVar`:

```
function requireEnvVar(envVarName: string): string {
   const result = process.env[envVarName];
   if (!result) { throw new Error('blah'); }
   return result;
}
```

2. Put an explicit type hint on the `const`, like `const signingEndpoint: string = process.env.MOMENTO_ENDPOINT`.  Not sure if the compiler would be satisfied with that one though.  The helper function is probably the better way to go.